packages/flame/example/lib/main.dart

```diff
@@ -14,14 +14,51 @@ void main() {
   );
 }
 
-class Square extends PositionComponent {
+/// This example simply adds a rotating white square on the screen, if you press
+/// somewhere other than on the existing square another square will be added and
+/// if you press on a square it will be removed.
+class MyGame extends FlameGame with DoubleTapDetector, HasTappables {
+  bool running = true;
+
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
```

Comment:
why do you need this, if super's `onLoad` is empty?

Reply:
It was already there, I just moved the class.
But I can remove it.

Reply:
Ah right, it has @mustCallSuper on it.

Reply:
well, the question then why must it, if the super doesn't do anything?

Reply:
I don't remember, there was some reasoning behind it though. Do you remember @erickzanardo or @luanpotter?

Reply:
The reason is some FlameGame and Component mixins do initialization onLoad. so if people don't call super, those are missed, causing bugs. Thus we added the annoation so people would not make such mistake.